modin/core/storage_formats/base/query_compiler.py

```diff
@@ -1246,6 +1284,28 @@ def prod(self, **kwargs):  # noqa: PR02
     def sum(self, **kwargs):  # noqa: PR02
         return DataFrameDefault.register(pandas.DataFrame.sum)(self, **kwargs)
 
+    @doc_utils.add_refer_to("DataFrame.mask")
+    def mask(self, cond, **kwargs):
+        squeeze_self = kwargs.pop("squeeze_self", False)
+
+        def mask(df, cond, **kwargs):
+            if squeeze_self:
+                df = df.squeeze(axis=1)
+            return df.mask(cond, **kwargs)
+
+        return DataFrameDefault.register(mask)(self, cond, **kwargs)
```

Comment:
Why can't we pass in `pandas.DataFrame.mask` here?

Reply:
We probably can, I think I've copy-pasted some of my other fix where the behaviour was different... I had already implemented the change locally before your comment, but haven't tested to it wasn't pushed. Should be fixed now.